api/v1alpha1/helper.go

```diff
@@ -106,10 +106,10 @@ func GetNamespaceName(etcdObjMeta metav1.ObjectMeta) types.NamespacedName {
 // to suspend spec reconciliation for this Etcd resource. If no annotation is set then it will return nil.
 func GetSuspendEtcdSpecReconcileAnnotationKey(etcdObjMeta metav1.ObjectMeta) *string {
 	if metav1.HasAnnotation(etcdObjMeta, SuspendEtcdSpecReconcileAnnotation) {
-		return pointer.String(SuspendEtcdSpecReconcileAnnotation)
+		return ptr.To(SuspendEtcdSpecReconcileAnnotation)
```

Comment:
Please look at all usages of `PointerOf` and `TypeDeref` and now you can replace these with `ptr.To` and `ptr.Deref` functions.

Reply:
Thanks for pointing this to me. I was not aware of this. 

Now I've addressed this concern. PTAL